osconfig_tests/main.go

```diff
@@ -85,23 +81,25 @@ func main() {
 		close(tests)
 	}()
 
-	var testSuites []*junitxml.TestSuite
 	for ret := range tests {
-		testSuites = append(testSuites, ret)
-	}
+		testSuiteOutPath := fmt.Sprintf("%s-%s", ret.Name, *outPath)
```

Comment:
outPath is supposed to be a path, maybe we need to change the flag to outdir? Then this would be filepath.Join

Reply:
yeah, changed the outdir.

Reply:
Also update config.yaml in test-infra/prow with this new flag

Reply:
done